x/crisis/autocli.go

```diff
@@ -0,0 +1,19 @@
+package crisis
+
+import (
+	autocliv1 "cosmossdk.io/api/cosmos/autocli/v1"
+	crisisv1beta1 "cosmossdk.io/api/cosmos/crisis/v1beta1"
+)
+
+// AutoCLIOptions implements the autocli.HasAutoCLIConfig interface.
+func (am AppModule) AutoCLIOptions() *autocliv1.ModuleOptions {
+	return &autocliv1.ModuleOptions{
+		Tx: &autocliv1.ServiceCommandDescriptor{
+			Service: crisisv1beta1.Msg_ServiceDesc.ServiceName,
+			// map v1beta1 as a sub-command
+			SubCommands: map[string]*autocliv1.ServiceCommandDescriptor{
+				"v1beta1": {Service: crisisv1beta1.Msg_ServiceDesc.ServiceName},
+			},
+		},
+	}
```

Comment:
@chandiniv1 I believe you are missing the tx command for crisis; in the case of crisis, there is only one `invariant-broken`

Reply:
if you are interested in solving this one, I can share other better example later today

Reply:
okay ,thank you

Reply:
@JeancarloBarrios Can you please share the example